packages/base/components/Related/RelatedCategoryTags.vue

```diff
@@ -0,0 +1,157 @@
+<script setup>
+const route = useRoute();
+const localePath = useLocalePath();
+
+const props = defineProps({
+  /**
+   * Array of tags
+   */
+  tags: {
+    type: Array,
+    required: true,
+  },
+  /**
+   * Array of tags selected by the user
+   */
+  selected: {
+    type: Array,
+    default: () => [],
+  },
+  /**
+   * Toggle to show or hide the heading
+   */
+  heading: {
+    type: Boolean,
+    default: true,
+  },
+  /**
+   * Scope of the categories
+   * @values stories, data-space
+   */
+  scope: {
```

Comment:
I think that naming this `routeName` would be clearer, and making it required instead of having a default.